packages/twenty-server/src/engine/metadata-modules/data-source/data-source.exception.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomError } from 'src/utils/custom-error';
+
+export class DataSourceException extends CustomError {
```

Comment:
shouldn't we call it CustomException? or TwentyException? 
not a big fan of an Exception extending an Error